jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/inputGroup/InputGroupCustomFileInputTests.java

```diff
@@ -0,0 +1,170 @@
+package io.github.epam.bootstrap.tests.composite.section.inputGroup;
+
+import io.github.epam.TestsInit;
+import org.openqa.selenium.By;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.driver.get.DriverData.PROJECT_PATH;
+import static com.epam.jdi.tools.PathUtils.mergePath;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.customFileInputPrepend;
+import static io.github.com.pages.BootstrapPage.customFileInputAppend;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Created by Olga Ivanova on 19.09.2019
+ * Email: olga_ivanova@epam.com
+ */
+
+public class InputGroupCustomFileInputTests extends TestsInit {
+    public final static String LABEL_TEXT = "Upload";
+    public final static String INPUT_TEXT_DEFAULT = "Choose file";
+    public final static String INPUT_TEXT = "general.xml";
+    public final static String FILE_NAME = "/src/test/resources/general.xml";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void labelTextTests() {
+        customFileInputPrepend.upload.is().text(LABEL_TEXT);
+        customFileInputPrepend.upload.is().value(LABEL_TEXT);
+
+        customFileInputAppend.upload.is().text(LABEL_TEXT);
+        customFileInputAppend.upload.is().value(LABEL_TEXT);
+    }
+
+    @Test
+    public void inputTextTests() {
+        customFileInputPrepend.label.is().text(INPUT_TEXT_DEFAULT);
+        customFileInputPrepend.label.is().value(INPUT_TEXT_DEFAULT);
+
+        customFileInputAppend.label.is().text(INPUT_TEXT_DEFAULT);
+        customFileInputAppend.label.is().value(INPUT_TEXT_DEFAULT);
+    }
+
+    @Test
+    public void uploadTest() {
+        customFileInputPrepend.click();
+        customFileInputPrepend.input.setValue(mergePath(PROJECT_PATH,FILE_NAME));
+        customFileInputAppend.click();
+        customFileInputAppend.input.setValue(mergePath(PROJECT_PATH,FILE_NAME));
+
+        customFileInputPrepend.label.is().text(INPUT_TEXT);
+        customFileInputPrepend.label.is().value(INPUT_TEXT);
+        customFileInputAppend.label.is().text(INPUT_TEXT);
+        customFileInputAppend.label.is().value(INPUT_TEXT);
+    }
+
+    @Test
+    public void isValidationOptionsSectionTests() {
+
+        //tests customFileInputPrepend element
```

Comment:
if you need a description for the test you can use or allure `@Description` annotation or description attribute of `@Test` annotation from TestNG. if this comment is not a test description, could you please tell me more about it